app/services/monthly_district_data_service.rb

```diff
@@ -101,7 +101,7 @@ def facility_rows
   end
 
   def common_attributes(region, follow_ups_by_month)
-    complete_monthly_registrations = repo.complete_monthly_registrations.find { |k, _| k.slug == region.slug }.last
```

Comment:
We no longer need to grab from `complete_monthly` numbers because we pass in the explicit range this service asks for.